playbooks/wordpress.yml

```diff
@@ -7,21 +7,21 @@
     - php_memory_ilimit: "128M"
     - php_max_execution_time: "90"
     - php_upload_max_filesize: "256M"
-    - php_version: "5.6"
+    - php_version: "7.3"
     - php_enable_php_fpm: true
     - php_webserver_daemon: "nginx"
-    - php_fpm_listen: "/var/run/php/php5-fpm.sock"
+    - php_fpm_listen: "/var/run/php/php7.3-fpm.sock"
     - php_packages:
-      - php-common
-      - php-fpm
-      - php-cli
-      - php-dev
+      - php7.3-common
+      - php7.3-fpm
+      - php7.3-cli
+      - php7.3-dev
       - libpcre3-dev
-      - php-gd
-      - php-curl
-      - php-imap
-      - php-json
-    - php_mysql_package: php5.6-mysql
+      - php7.3-gd
+      - php7.3-curl
+      - php7.3-imap
+      - php7.3-json
+    - php_mysql_package: php7.3-mysql
   roles:
```

Comment:
@giovannicimolin, we don't have the backups enabled for the VM this playbooks runs on. You will have to set and specify the `TARSNAP_BACKUP_FOLDERS`  variable to get it to work. You should backup at least the `/var/www`, `/etc/nginx/sites-available` directories and also create a snitch and configure it in the `ansible-secrets` playbook after configuring all the `TARSNAP_*` variables.

Reply:
@lgp171188 Setting up backups is not in the scope of this ticket.